tests/test_basic.py

```diff
@@ -1116,6 +1116,25 @@ def test_static_files():
     rv.close()
 
 
+def test_static_path_depricated():
```

Comment:
That's what I thought and how I typed it until I used it from this doc:
http://pytest.org/latest/recwarn.html#ensuring-a-function-triggers-a-deprecation-warning

and the python builtin spells it with an e. So I'm fine either way. Do you need me to push it again or is this something that you were able to fix/change if you need?